### PR TITLE
Adding scripting examples relevant for production testing  

### PR DESCRIPTION
**Pull request by jovatn**
## Description

Adding script examples, primarily for production tests
    
 - reading out eeprom/vpd data (sysrepocfg)
 - LED testing (shell/echo)
 - power supply sensor testing (shell/cat)
 - Ethernet port test - example for looping ports (CLI setup)

A reference to a production test script for the first three items would be great extension to this.
    
The 4th item (Ethernet port configuration for looping) should perhaps be documented elsewhere.
Maybe in Networking Lego page?

## Checklist

Tick relevant boxes, this PR is-a or has-a:

- [ ] Bugfix
  - [ ] Regression tests
  - [ ] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [X] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

**Review by troglobit — CHANGES_REQUESTED** [review]
A few comments and suggestions for improvement.  Please note, the suggestions for *you* form changes apply to previous additions already in this document.

Also, I noticed a badly formatted internal reference in https://github.com/kernelkit/infix/blob/main/doc/scripting.md#check-devices-network-connectivity that you perhaps can have a look at when you're at it?

File: doc/scripting.md [hunk 1]
```diff
@@ -331,6 +331,65 @@ admin@infix.local%eth0's password: *****
 ~$ 
 ```
 
+### Controlling LEDs (For Production Tests)
+
+As part of production tests you wish to verify that LEDs work as
+expected. Infix uses standard [Linux support for LED
```
[the remark this answers is not in the document]
Comment (jovatn) [reply doc/scripting.md:337]: Agreed

File: doc/scripting.md [hunk 2]
```diff
@@ -331,6 +331,65 @@ admin@infix.local%eth0's password: *****
...
+~$ ssh admin@example.local 'echo 1 | sudo tee /sys/class/leds/red\:lan/brightness'
+~$ 
+```
+
+To turn off the same LED, run the following commands.
+
+```
+~$ ssh admin@example.local 'echo none | sudo tee /sys/class/leds/red\:lan/trigger'
+~$ ssh admin@example.local 'echo 0 | sudo tee /sys/class/leds/red\:lan/brightness'
+~$
```
[the remark this answers is not in the document]
Comment (jovatn) [reply doc/scripting.md:364]: Good practice!

File: doc/scripting.md [hunk 3]
```diff
@@ -688,7 +747,405 @@ An alternative is to write it to a temporary file, and use `sysrepocfg
...
+| VLAN 20: e3 & e4  | e4 <=> e5 |
+| VLAN 30: e5 & e6  | e6 <=> e7 |
+| VLAN 40: e7 & e8  | e8 <=> e9 |
+| VLAN 50: e9 & e10 |           |
+
+The test PC is connected to e1 and e10 via different interfaces
+(alternatively, two different PCs are used).
+
+> Configuration here is done via console. If you intend to do it via
+> Ethernet and SSH, be careful so that you do not lose
```
[the remark this answers is not in the document]
Comment (jovatn) [reply doc/scripting.md:821]: I updated the text, with a mix of the old a new, as I have not used e10 for configuration.
Perhaps the example should be updated to use SSH.

**Review by troglobit — APPROVED** [review]
Looks great, approved!

### Minor Comments

For future reference, we try to verbatim format UNIX paths, e.g., `/sys/class/leds/`, and use trailing slash when the destination is a directory.

Also, did you see my comment about ` & ` vs `, ` in the VLAN snake table?

**Comment by troglobit** [comment]
Since this has been approved, you  can merge this at your own discretion.

**Pull request merged**